.github/pull_request_template.md

```diff
@@ -1,18 +1,8 @@
-<!-- Thank you for your pull request to AMReX!
-
-  1. Be sure the title above is descriptive
-  2. Provide a summary of the proposed changes
-  3. Provide any background to help reviewers understand the proposed changes
-  4. Fill out the checklist (using [x] to check the box)
-
-NOTE: All text between < and > will not be formatted in md previewers
--->
-
 ## Summary
 
 ## Additional background
 
-## Checklist
+## Checklist <-- use [x] to check the box -->
```

Comment:
What do you think about removing the html comment here too or moving to the top of the template?  The reason for it is I use a firefox extension for this and it only removes html comment if it's at the beginning, because it is not always safe to do it in the middle.

Reply:
Ha!  The world's most PR'd 3 lines of text!  I say through that away too.